markdown/spring-cloud-stream-basics/spring-cloud-stream-basics.md

```diff
@@ -173,12 +173,16 @@ If you look at the `pom.xml` file you'll see a few important things:
 - The java version you chose is included
 - The "Spring Cloud Stream" and "Spring Cloud Starter Stream Solace" dependencies are included
 
-![Pom File](img/pom.webp)
+![Pom File](img/pom.jpg)
 
 ### Add Messaging Service Connection Info
 
 Open the application config file under `src/main/resources` and enter the properties below substituting the connection information with your messaging services' connect info we got from the Solace Cloud Connect tab in the previous section.
 
+### <strong>A. PubSub+ Broker running locally in Docker</strong>
```

Comment:
using a `<strong>` would still work for bolding; for future reference you can also use `**` as follows
```
### **A. PubSub+ Broker running locally in Docker**
```

Reply:
Sure!